src/test/java/com/wikia/webdriver/testcases/mercurytests/curatedcontenttests/NavigationTests.java

```diff
@@ -116,29 +121,74 @@ public void MercuryCuratedNavigationTest_003_navigateThroughNamespaces() {
+    String
+        expectedUrl =
+        urlBuilder.getUrlForPathWithoutWiki(Configuration.getWikiName(),
+                                            MercurySubpages.CC_CATEGORY_TEMPLATES);
     String testUrl = expectedUrl;
     section.openWikiPage(testUrl);
     section.waitForLoadingSpinnerToFinish();
     Assertion.assertUrlEqualToCurrentUrl(driver, expectedUrl);
 
-    expectedUrl = urlBuilder.getUrlForPathWithoutWiki(Configuration.getWikiName(), MercurySubpages.CC_SECTION_CATEGORIES);
+    expectedUrl =
+        urlBuilder.getUrlForPathWithoutWiki(Configuration.getWikiName(),
+                                            MercurySubpages.CC_SECTION_CATEGORIES);
     testUrl = expectedUrl;
     section.openWikiPage(testUrl);
     section.waitForLoadingSpinnerToFinish();
     Assertion.assertUrlEqualToCurrentUrl(driver, expectedUrl);
 
-    expectedUrl = urlBuilder.getUrlForPath(Configuration.getWikiName(), MercurySubpages.CC_MAIN_PAGE);
-    testUrl = urlBuilder.getUrlForPathWithoutWiki(Configuration.getWikiName(), MercurySubpages.CC_EMPTY_CATEGORY);
+    expectedUrl =
+        urlBuilder.getUrlForPath(Configuration.getWikiName(), MercurySubpages.CC_MAIN_PAGE);
+    testUrl =
+        urlBuilder.getUrlForPathWithoutWiki(Configuration.getWikiName(),
+                                            MercurySubpages.CC_EMPTY_CATEGORY);
     section.openWikiPage(testUrl);
     section.waitForLoadingSpinnerToFinish();
     Assertion.assertUrlEqualToCurrentUrl(driver, expectedUrl);
 
-    expectedUrl = urlBuilder.getUrlForPath(Configuration.getWikiName(), MercurySubpages.CC_MAIN_PAGE);
-    testUrl = urlBuilder.getUrlForPathWithoutWiki(Configuration.getWikiName(), MercurySubpages.CC_NOT_EXISTING_SECTION);
+    expectedUrl =
+        urlBuilder.getUrlForPath(Configuration.getWikiName(), MercurySubpages.CC_MAIN_PAGE);
+    testUrl =
+        urlBuilder.getUrlForPathWithoutWiki(Configuration.getWikiName(),
+                                            MercurySubpages.CC_NOT_EXISTING_SECTION);
     section.openWikiPage(testUrl);
     section.waitForLoadingSpinnerToFinish();
-    section.isAlertNotificationVisible();
+    MercuryAlertComponentObject mercuryError = new MercuryAlertComponentObject(
+        driver, Mercury.AlertMessage.NOT_EXISTING_SECTION);
+
+    Assertion.assertTrue(mercuryError.isAlertMessageVisible());
     Assertion.assertUrlEqualToCurrentUrl(driver, expectedUrl);
   }
+
+  @Test(groups = "MercuryCuratedNavigationTest_005")
+  @Execute(onWikia = MercuryWikis.MERCURY_CC)
+  public void MercuryCuratedNavigationTest_005_redirectToExistingArticle() {
+    ArticleContent content = new ArticleContent();
+    String redirect = WikiTextContent.REDIRECT + WikiTextContent.LINK_PREFIX +
+                      MercurySubpages.CC_REDIRECT_DESTINATION + WikiTextContent.LINK_POSTFIX;
+    content.push(redirect, MercurySubpages.CC_REDIRECT_SOURCE_1);
```

Comment:
there is only one operation on this variable, please use:
`new ArticleContent().push(redirect, MercurySubpages.CC_REDIRECT_SOURCE_1);`